src/obfuscation.ts

```diff
@@ -12,3 +14,22 @@ export function encodeBase64(input: string) {
 export function decodeBase64(input: string) {
   return base64.atobPolyfill(input);
 }
+
+export function obfuscatePrecomputedFlags(
+  precomputedFlags: Record<string, PrecomputedFlag>,
+): Record<string, PrecomputedFlag> {
+  const response: Record<string, PrecomputedFlag> = {};
+  Object.keys(precomputedFlags).map((flagKey) => {
+    const assignment = precomputedFlags[flagKey];
+
+    response[getMD5Hash(flagKey)] = {
+      variationType: assignment.variationType,
+      extraLogging: assignment.extraLogging,
```

Comment:
Extra logging should be base64-encoded, similar to what we do in UFC

Reply:
done